recipes/7bitconf/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,7 @@
+#include <SevenBit/Conf/LibraryConfig.hpp>
+#include <iostream>
+
+int main() {
+  std::cout << "7bitconf version: " << _7BIT_CONF_VERSION;
```

Comment:
```suggestion
  std::cout << "7bitconf version: " << _7BIT_CONF_VERSION;
  IConfiguration::Ptr configuration = ConfigurationBuilder{} //
                                            .addAppSettings()
                                            .addEnvironmentVariables()
                                            .addCommandLine(argc, argv)
                                            .build();
```

Try to consume something that requires the library to validate the linkage too.